mne/source_space.py

```diff
@@ -3197,3 +3198,65 @@ def _get_src_nn(s, use_cps=True, vertices=None):
     else:
         nn = s['nn'][vertices, :]
     return nn
+
+
+@verbose
+def compute_distance_to_sensors(src, info, picks=None, trans=None,
+                                verbose=None):
+    """Compute distances between vertices and sensors.
+
+    Parameters
+    ----------
+    src : instance of SourceSpaces
+        The object with vertex positions for which to compute distances to
+        sensors.
+    info : instance of Info
+        Measurement information with sensor positions to which distances shall
+        be computed.
+    %(picks_good_data)s
+    %(trans_not_none)s
+    %(verbose)s
+
+    Returns
+    -------
+    depth : array of shape (n_vertices, n_channels)
+        The Euclidean distances of source space vertices with respect to
+        sensors.
+    """
+    from scipy.spatial.distance import cdist
+
+    assert isinstance(src, SourceSpaces)
+    _validate_type(info, (Info,), 'info')
+
+    # Load the head<->MRI transform if necessary
+    if src[0]['coord_frame'] == FIFF.FIFFV_COORD_MRI:
+        src_trans, _ = _get_trans(trans, allow_none=False)
+    else:
+        src_trans = Transform('head', 'head')  # Identity transform
+
```

Comment:
@olafhauk to cover this line and test without passing a trans you should pass the src from a forward instance `fwd['src']` or `inv['src']`.

HTH

Reply:
That's what I tried on ll. 104-6 in test_compute_distance_to_sensors() (see comments in code), but the tests fail.